src/components/CImage/index.meta.js

```diff
@@ -44,13 +46,23 @@ export default {
       name: 'Data Source',
       value: null,
       schema: itemInterface,
+      disabled: {
+        current: false,
+        default: false,
+        expression: binding.setExpression('<%= element.src != null && element.src.length > 0 %>'),
```

Comment:
Use strict comparison `!==` or `isNil` lodash method here (same is on several places).